guide/payments/units-and-symbols.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: guide
+title: Units & Symbols
+nav_order: 1
+parent: Payments
+permalink: /guide/payments/units-and-symbols/
+typora-root-url: ./images
+---
+
+# Units and symbols
+
+<!--Don't forget to update date -->
+11 December 2020
+
+Bitcoin, bits , sats. The format and presentation of bitcoin values are probably amongst the most challenging for new users. The objective of this section is to provide observations of its current usage and recommendations for the best option if multiple formats exist. The goal is not to re-invent the format but to observe the current usage, and eventually to codify it through an organic approach. The surest way to achieve wide-scale adoption is to standardize what is already being used today. 
+
+
+
+#### Objective
+
+Despite the challenge of the fractional aspect and the abundance of digits used to present the value of bitcoin, the objective is to serve as a guide to format the values so they are easily scannable, quick to read, and possible to remember. It's a tall order given the challenge, but that will be the horizon to aim. 
+
+
+
+#### Current adoption
+
+The most common way bitcoin is expressed is BTC (bitcoin)  with 8 decimal places. While not as common, denominations of BTC such as mBTC ("millibitcoins"), μBTC ("bits") and sat ("satoshi") are also used. The chart below illutrates how each units relate to the BTC unit.
+
+| Unit     | Symbol       | Example (English) | Example (Non-English) |
```

Comment:
I think a conversion table would be more appropriate for this page, as this current one touches on the amount display. If you are going to touch amount display I'd put it under another heading to discuss the locales of thousand and decimal separators.

Reply:
Is this feedback still relevant after the changes I made last week?